core-bundle/assets/controllers/checkbox-controller.js

```diff
@@ -0,0 +1,70 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class extends Controller {
+    static targets = ['checkAll', 'input'];
+
+    static values = {
+        rowGuards: {
+            type: Array,
+            default: ['input', 'label', 'button', 'a', '.operations'],
+        },
+    };
+
+    initialize() {
+        this.start = null;
+    }
+
+    rowClick(event) {
+        if (event.target.closest(this.rowGuardsValue)) {
+            return;
+        }
+
+        const clickTargets = event.currentTarget.querySelectorAll('input[type=checkbox],input[type=radio]');
```

Comment:
can you explain why it is `event.currentTarget` here but `event.target` above? (I never understood the difference 🙈)

Reply:
also, can there / should there ever be more than one target, or can we use `querySelector` here?

Reply:
Kinda done here: 04fda94d06eca4eec58b36d4ad1079125d6ed46d, deleted a line but readded in 84a62747ecfcd8845cf4034d0350c7d282cb967e